src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/stores/PreviewStore.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+import queryString from 'query-string';
+import {action, computed, observable} from 'mobx';
+import Requester from 'sulu-admin-bundle/services/Requester';
+import type {PreviewRouteName} from './../types';
+
+const generateRoute = (name: PreviewRouteName, options: Object): string => {
+    const query = queryString.stringify(options);
+    if (query.length === 0) {
+        return PreviewStore.routes[name];
+    }
+
+    return PreviewStore.routes[name] + '?' + query;
+};
+
+export default class PreviewStore {
+    static routes: { [PreviewRouteName]: string } = {};
```

Comment:
We don't do padding in the types if they are written in a single line. Looks like we only have an eslint rule for ordinary object and not flow types :confused: